demo/Signs.swift

```diff
@@ -590,8 +590,116 @@ extension Sign {
                 return nil
             case .warningSecondRoadLeft:
                 return nil
+            case .regulatoryNoTurnOnRedText:
```

Comment:
I think we should use `default` for US, it has no such diversity of signs, .US classifier can detect just 77 classes

Reply:
Please see comment above, potential issue here is the same.
As a general solution in scope of https://github.com/mapbox/mapbox-vision-ios/issues/122 I think we can try to introduce smth like a platform-specific wrapper to have an ability to control signs for each country.

Reply:
Agree, we could make it more neat with wrapper or smth like that but for now it can and should be done in much easier way, for my opinion

Reply:
Update: squash nil-cases under common `default`.